content/api/events-ingest.apib

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+title: Ingest
+description: Send event data to SparkPost for processing
+label: New
+
+# Group Ingest
+
+The Events Ingest API provides an interface for on-premise MTA users to quickly, easily, and safely
+take advantage of SparkPost's cloud-specific services.
+The Events Ingest API accepts email event data, normalizes it, and sends it through SparkPost's data pipeline until it is ultimately consumable by various analytical services. 
+
+### Event Format
+
+Events sent to the Ingest API must adhere to a specific format in order to be correctly processed by SparkPost.
+Only certain aspects of incoming API requests are validated during the REST transaction. Requests that do not pass validation at this point are rejected and a HTTP error code will be returned.
+The body of the request must contain gzip-encoded, [new-line-delimited](http://ndjson.org/) JSON along with the appropriate headers.
+The size of the request is limited to 5 MB.
+
+Additional validation occurs out of band of the request. This additional validation ensures that events specified as part of the request body
+are properly formatted and consumable by SparkPost analytical services. Events that are not proper JSON, lack the correct wrapper, or are missing required fields will be dropped. 
+Please see [Processing Out of Band Errors](#header-processing-out-of-band-errors) for additional information.
+
+
+### Processing Out of Band Errors
+
+**Coming Soon**
+
+## Ingest [/v1/ingest]
+
+### Send Events [POST /v1/ingest/events]
+
+Post a new-line delimited list of JSON events.
+
++ Data Structure
+    + msys (object, required) - The event wrapper
+        + [event grouping] (object, required) - Designates the event grouping <br />One of: `message_event`, `track_event`, `gen_event`, `adaptive_event`, `unsubscribe_event`, `relay_event`, `ab_test_event`
```

Comment:
![screen shot 2019-02-22 at 11 09 27 am](https://user-images.githubusercontent.com/7385860/53255001-ad2c5680-3692-11e9-9265-af1cda703e3e.png)

Reply:
What do brackets mean? `[event grouping]`

If object, then why does transport method get one?

Reply:
Also all of the grouping types are listed, should we list all of the event types too?

Reply:
I used brackets to signify "an object key is needed here, but it's one of the following..." . I'm not sure how to signify an object key without giving the exact name of the key, like `type`.

The grouping types _have_ to be one of those listed; however, the value of `type` is not currently validated - we're just checking that it's there. Is there a definitive list of event types? Or I could list some examples?

Reply:
Ok, looks like I want Italics and not brackets ref: https://github.com/apiaryio/mson#variable-property-name